config/example/config.txt

```diff
@@ -385,4 +385,7 @@ HIGH_POP_MC_MODE_AMOUNT 65
 DISABLE_HIGH_POP_MC_MODE_AMOUNT 60
 
 ##Uncomment to enable developer start. Auto starts the server after initialization
-##DEVELOPER_EXPRESS_START
\ No newline at end of file
+##DEVELOPER_EXPRESS_START
+
+##Uncomment to have the lobby sound default to off for new players, useful for testing servers without database.
+LOBBY_SOUND_OFF
```

Comment:
Comment it out as default.

Reply:
woops, forgot to do that when I copied it over from my config.txt